packages/react-theme/src/themes/ColorRamp.stories.tsx

```diff
@@ -6,6 +6,8 @@
 // @ts-nocheck
 
 import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
```

Comment:
AFAIK we only need this for SSR, is our storybook server rendered ?

Reply:
no plans with SSR, but we have a lint rule which forbids `useLayoutEffect`. I though that it is less harm to just stick to `useIsomorphicLayoutEffect` here to build a muscle memory than lint-ignoring the `useLayoutEffect`.

Reply:
Yeah I don't mind, just confused... because you are lint ignoring extraneous dependencies at this point 😅😅

Reply:
That's fair, but stories are supposed to be able to import anything without polluting real package's dependencies. 
cc @Hotell

Reply:
hmm I'd prefer to disable that rule and use native react apis. This might be confusing to contributors (as you both mentioned)
- what does this function do in particular for storybook ?
- hmm SSR? we/they do SSR with storybook ? hmm 

action point ? 
- that eslint rule should be disabled for stories

WDYT?

Reply:
If you are good with lint rules for stories being different to rules for production code that works for me the best.